sandbox-template.md

```diff
@@ -0,0 +1,162 @@
+# CNCF SIG Sandbox Project Request Template & Promotion Template
+
+
+# <Project> 
+
+
+## <Promotion/Acceptance> to CNCF <Incubation/Sandbox>
+
+**Primary aim: This template helps a project to prepare for a presentation to the SIG, and then helps the SIG share the collated information with the TOC.**
+
+**Authors:**
+
+
+# Background
+
+**_Link to TOC PR_**
+
+**_Link to Presentation_**
+
+**_Link to GitHub project_**
+
+
+## Project Goal
+
+
+## Current Status
+
+
+## Future Plans
+
+
+# Project Scope
+
+
+## Clear project definition
+
+**_Does the project have a clear and well defined scope_**
+
+## Value-add to the CNCF ecosystem
+
+**_Does the project have a clear value add to the current project ecosystem. How does it relate to other projects with overlapping capabilities._**
+
+
+## Alignment with other CNCF projects
```

Comment:
Should we also include some other projects outside CNCF?
For example, any other similar projects?

Reply:
+1 yes please